css/style.css

```diff
@@ -1,4 +1,538 @@
+  text-transform: uppercase;
+  color: #e51937;
+  padding: 10px 0 0 0;
+}
+
+.btn__test {
+  color: #e51937;
+  border: 2px solid #e51937;
+}
+
+.btn__test:hover {
+  background-color: #e51937;
+  color: #fff;
+  -webkit-transition: ease-in-out .5s;
+  -o-transition: ease-in-out .5s;
+  transition: ease-in-out .5s;
+}
+
+.button {
+  margin: 5px 0 0;
+}
+
+.test__img {
+  display: block;
+  margin: 5px  auto 0;
+  width: 100%;
+  height: 100%;
+  -ms-flex-item-align: start;
+  align-self: flex-start;
+}
+
+.test__arrow {
+  margin: 39px auto 0;
+  width: 30px;
+}
+
+@media (max-width: 768px) {
+  .header__right_row{
+    display: -webkit-flex;
+    display: -moz-flex;
+    display: -ms-flex;
+    display: -o-flex;
+    display: flex;
+    width: 35px;
+    justify-content: flex-end;
+  }
+
+  .menu__item,
+  .menu__row .menu__item:first-child {
+    display: none;
+  }
+
+  .test{
+    margin: 40px 0 0;
+  }
+
+  .test__row {
+    margin: 0 auto;
+    -ms-flex-wrap: wrap;
+    flex-wrap: wrap;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    max-width: 490px;
+    margin: 0 auto;
+    padding: 0;
+  }
+
+  .test__row_right {
+    -webkit-box-flex: 0;
+    -ms-flex: 0 1 100%;
+    flex: 0 1 100%;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    max-width: 488px;
+    padding: 0;
+    margin: 0 auto;
+  }
+
+  .test__row_left {
+    -webkit-box-flex: 0;
+    -ms-flex: 0 1 418px;
+    flex: 0 1 418px;
+    -webkit-box-pack: center;
+    -ms-flex-pack: center;
+    justify-content: center;
+    padding: 0;
+    margin: 55px auto 0;
+    max-width: 418px;
+  }
+
+  .test__column {
+    -webkit-box-flex: 0;
+    -ms-flex: 0 1 50%;
+    flex: 0 1 50%;
+    margin: 0;
+  }
+
+  .test__body_testdrive {
+    background: url(../img/icons/steering-wheel_red.png) no-repeat;
+    font-size: 16px;
+    color: red;
+  }
+
+  .test__body {
+    font-size: 50px;
+  }
+
+  .btn__test {
+    width: 220px;
+    font-size: 16px;
+    background-color: #e51937;
+    color: #fff;
+  }
+
+  .test__img_speed {
+    width: 65.92px;
+    height: 48.91px;
+  }
+
+  .test__column_testdrive {
+    padding: 0;
+  }
+
+  .test__img_road {
+    width: 40px;
+    height: 70px;
+  }
+
+  .test__img_wheel {
+    width: 31px;
+    height: 31px;
+  }
+
+  .test__arrow {
+    margin: 64px auto 0;
+   }
+ }
+
+@media (max-width: 375px) {
+  .logo {
+    max-width: 166px;
+    max-height: 42px;
+  }
+
+  .main-title {
+    font-size: 50px;
+  }
+
+  .banner__text {
+    font-size: 10px;
+  }
+
+  .test{
+    margin: 18px auto 0;
+    max-width: 305px;
+  }
+
+  .test__row_right {
+    max-width: 284px;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+  }
+
+  .test__row_left {
+    max-width: 303px;
+    margin: 46px auto 0;
+  }
+
+  .test__body {
+    font-size: 27px;
+  }
+
+  .btn__test {
+    font-size: 13px;
+    background-color: #E51937;
```

Comment:
Codacy found an issue: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/chendler_tesla/pullRequest?prid=5803911)